### DISCUSSION TITLE
Can not execute `setup.py`

### DISCUSSION THREAD

**Discussion by ghost** [mir-group/nequip#258] [General]
Hi, =)
Can not execute `setup.py` since setuptools is not available in the build environment.

When I tried to compile files I encountered this message and solved it with just typing
"pip install --force-reinstall setuptools"

Hopefully it worked but, a wheel could not be built and nequip was installed using 'setup.py install' method.
Will it cause any problem?
Thanks. =)

**Comment by Linux-cpp-lisp** [comment]
Hi @gud5201 ,

Thanks for your interest in our code!

This sounds like an issue with your environment; searching the error reveals various possible causes but none seem to be related to the package itself. Please let us know if you continue to have issues that seem to be unique to or directly caused by the `nequip` packaging / our install instructions.
